google-cloud-storage/samples/acceptance/helper.rb

```diff
@@ -0,0 +1,89 @@
+# Copyright 2020 Google, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "google/cloud/errors"
+require "google/cloud/kms"
+require "google/cloud/storage"
+require "minitest/autorun"
+require "net/http"
+require "securerandom"
+require "uri"
+
+def create_bucket_helper bucket_name
+  storage_client = Google::Cloud::Storage.new
+
+  retry_resource_exhaustion do
+    return storage_client.create_bucket bucket_name
+  end
+end
+
+def delete_bucket_helper bucket_name
+  storage_client = Google::Cloud::Storage.new
+
+  retry_resource_exhaustion do
+    bucket = storage_client.bucket bucket_name
+    return unless bucket
+
+    bucket.files.each(&:delete)
+    bucket.delete
+  end
+end
+
+def retry_resource_exhaustion
+  5.times do
+    begin
+      yield
+      return
+    rescue Google::Cloud::ResourceExhaustedError => e
+      puts "\n#{e} Gonna try again"
+      sleep rand(3..5)
+    rescue StandardError => e
+      puts "\n#{e}"
+      return
```

Comment:
Why rescue and puts `StandardError` here?

Reply:
debug thing I forgot to remove. removed.